database/postgres/database.dbml

```diff
@@ -0,0 +1,107 @@
+// country, state, city is mandatory only if customer is buying an artwork.
+// Address fields will be filled accordingly based on whether customer is residing in personal
+// house, or flat (or society). Addresses not mandatory unless customer is buying an artwork.
+Table artbase.customer {
+  id int [primary key, increment]
+  name varchar [not null]
+  country varchar
+  state varchar
+  city varchar
+  home_address varchar [note: 'House number, street name, etc.']
+  apartment_name varchar
+  flat_number varchar
+  floor varchar
+  tower varchar
+  phone_number long [note: 'Phone number is not mandatory unless they are buying artwork']
```

Comment:
The note part is not required as it is applicable to all other fields except the ones which are not null.